### PR TITLE
Tools: speed up Carbonix build script

### PR DESCRIPTION
**Pull request by robertlong13**
SW-420

### CODE REVIEW

File: libraries/AP_HAL_ChibiOS/hwdef/CarbonixCommon/aircraft_configuration/Volanti_AC_1.xml [hunk 1]
```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0"?>
+<!-- "cx_pilot_commit_id" will be replaced by the actual commit id during the build process -->
+<data>
+    <aircraft>
+        <model>Volanti</model>
+        <model_version>AC_1</model_version>
+        <model_description>This is CubOrange, Diagonal CAN Architecture with Non Crystal CPN</model_description>
```
Comment (robertlong13) [inline libraries/AP_HAL_ChibiOS/hwdef/CarbonixCommon/aircraft_configuration/Volanti_AC_1.xml]:
```suggestion
CubeOrange, Diagnonal CAN Architecture, Non-Crystal CPNs
```

File: libraries/AP_HAL_ChibiOS/hwdef/CarbonixCommon/aircraft_configuration/Volanti_AC_2.xml [hunk 2]
```diff
@@ -43,10 +43,20 @@
...
+        <cpn id="29">
+            <bus>1</bus>
+            <firmware_name>gps_org.ardupilot.HolybroG4_GPS</firmware_name>
+            <firmware_commit>6e519848</firmware_commit>
+        </cpn>
         <cpn id="101">
             <bus>1</bus>
-            <firmware_name>arspd</firmware_name>
-            <firmware_commit>cx_pilot_commit_id</firmware_commit>
+            <firmware_name>arspd_org.ardupilot.f405_MatekGPS</firmware_name>
```
Comment (robertlong13) [inline libraries/AP_HAL_ChibiOS/hwdef/CarbonixCommon/aircraft_configuration/Volanti_AC_2.xml]: Should we add copies of these binaries into our repo?

**Review by robertlong13 — COMMENTED** [review]
LGTM.

@loki077 can give it the green check for me since github counts this as my PR (since I opened it way back when it was a smaller scope)

**Review by loki077 — APPROVED** [review]
LGTM - skipping the changes which cause long job name.

**Pull request merged**